### PR TITLE
GH-2524 - Updating menu icon and placement

### LINKED ISSUE #2524: UX: Missing icon for "New template from card" and "New template from board"
## Steps to reproduce the behavior

1. Open a card
2. Select  the menu (...)
3. "New template from card" has no icon
4. Select the board icon from the sidebar
5. "New template from board" has no icon

## Expected behavior

Icons for all menu items (or at least, menu item text should be aligned)

## Screenshots

Screen Shot 2022-03-10 at 9 37 08 PM

## Edition and Platform

 - Edition: Personal Desktop / Personal Server / Mattermost plugin
 - Version: 0.15 RC1
 - Browser and OS: Chrome on Mac

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
GH-2524 - Updating menu icon and placement

#### Ticket Link
Fixes #2524

### CODE REVIEW

File: webapp/src/widgets/menu/menu.tsx [hunk 1]
```diff
@@ -31,9 +32,9 @@ export default class Menu extends React.PureComponent<Props> {
     }
 
     public render(): JSX.Element {
-        const {position, children} = this.props
+        const {position, fixed, children} = this.props
         return (
-            <div className={'Menu noselect ' + (position || 'bottom')}>
+            <div className={`Menu noselect ' ${position || 'bottom'} ${fixed && ' fixed'}`}>
```
Comment (sbishel) [inline webapp/src/widgets/menu/menu.tsx:37]: Appears to be some mixed quotation marks....see the unit tests.

**Review by sbishel — CHANGES_REQUESTED** [review]
The update snapshots are not correct. I believe there are some mixed quotation marks causing these changes.

File: webapp/src/widgets/menu/menu.tsx (hunk 1 above)
Comment (asaadmahmood) [reply webapp/src/widgets/menu/menu.tsx]: @sbishel Should be fixed.

**Review by jespino — APPROVED** [review]
LGTM, but you need to resolve the merge conflict.

**Pull request merged**